boefjes/boefjes/plugins/kat_maxmind_geoip/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "title": "Arguments",
+  "type": "object",
+  "properties": {
+    "MAXMIND_USER_ID": {
+      "title": "User ID",
+      "type": "string",
+      "maxLength": 2048,
+      "description": "User id for MaxMind"
+    },
+    "MAXMIND_LICENCE_KEY": {
+      "title": "Licence Key",
+      "maxLength": 2048,
+      "type": "string",
+      "description": "Licence key for MaxMind"
+    }
```

Comment:
Could we also add the cache duration, the timeout, and the source url's in here to be configured by the user?

Reply:
https://github.com/minvws/nl-kat-coordination/issues/3301